scripts/prettier/index.js

```diff
@@ -33,16 +33,27 @@ const defaultOptions = {
 };
 const config = {
   default: {
-    patterns: ['src/**/*.js', 'www/**/*.js'],
-    ignore: [
-      '**/third_party/**',
-      '**/node_modules/**',
-      '**/jsfiddle-integration*.js',
+    patterns: [
+      // Internal forwarding modules
+      'packages/*/*.js',
+      // Source files
+      'packages/*/src/**/*.js',
```

Comment:
~~Should we also include `packages/*/npm/*.js` to ensure the NPM entry points stay formatted?~~

Nevermind. Looks like these are covered below in `scripts`